docs/src/main/asciidoc/optaplanner.adoc

```diff
@@ -0,0 +1,1034 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+= OptaPlanner - Using AI to optimize a schedule with OptaPlanner
+
+include::./attributes.adoc[]
+:config-file: application.properties
+
+This guide walks you through the process of creating a https://quarkus.io/[Quarkus]
+application with https://www.optaplanner.org/[OptaPlanner]'s constraint solving Artificial Intelligence (AI).
+
+== What you will build
+
+You will build a REST application that optimizes a school timetable for students and teachers:
+
+image::images/optaplanner/timeTableAppScreenshot.png[]
```

Comment:
I'm not sure this will work on the website. Could you follow how it's done in other guides? See the Kafka one for instance.

Reply:
Ah, the issue is not the subdirectory, but the "images/" prefix? removing that

Reply:
I'll avoid the subdirectory for now too, just so it's like the others